.github/workflows/publish-e2e-containers.yaml

```
@@ -0,0 +1,87 @@
+      - main
+
+env:
+  QUAY_ORG: quay.io/flightctl
+  # Enable BuildKit for cache mounts
+  DOCKER_BUILDKIT: 1
+  BUILDKIT_PROGRESS: plain
+
+jobs:
+
+
+
```

YAMLlint (1.37.1)

[warning] too many blank lines (3 > 2)

(empty-lines)